frontend/README.md

```diff
@@ -95,31 +95,33 @@ npm --prefix frontend run cypress:cp
 ### E2E Tests
 
 #### Requirements
-- Three shell terminals
 - Starting directory is the project root
-- Podman, Docker
-- NodeJS 22+
+- Podman/Docker Compose
 
-Start CPT-Dashboard backend data service end-to-end test dependency.
+Enable Podman Socket to facilitate Podman Compose
 ```shell
-./backend/e2e_backend.sh
+systemctl --user enable --now podman.socket         
 ```
 
-Start frontend GUI server and execute end-to-end tests.
+Generate a CPT Dashboard version file.
+```shell
+./backend/scripts/version.py        
+```
+Execute the end-to-end tests.
 ```shell
-npm --prefix frontend run cypress:run:ci
+./frontend/tests/e2e.sh
```

Comment:
The version generation should be built into the build scripts, not executed manually.

Reply:
oooooooooooooooh i got you